code/controllers/subsystem/mapping.dm

```diff
@@ -844,6 +848,22 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 
 			true_to_offset_planes[string_real] |= offset_plane
 
+/datum/controller/subsystem/mapping/proc/lazy_load_template(template_key)
+	if(LAZYACCESS(loaded_lazy_templates, template_key))
+		return
+	LAZYADDASSOC(loaded_lazy_templates, template_key, TRUE)
+
+	var/obj/effect/landmark/lazy_template_pivot/found
+	for(var/obj/effect/landmark/lazy_template_pivot/lazy_template_pivot in GLOB.lazy_template_pivots)
+		if(lazy_template_pivot.key != template_key)
+			continue
+		found = lazy_template_pivot
+		break
+
```

Comment:
would yeet all of this

Reply:
that or this should be a "load each of these pivots" pattern, rather then loading just one